src/pages/public_registration/ActionPanelContent.tsx

```diff
@@ -98,6 +133,24 @@ export const ActionPanelContent = ({
           />
         </ErrorBoundary>
       )}
+      <Box sx={{ m: '0.5rem', mt: '1rem' }}>
+        <Button fullWidth variant="contained" onClick={() => onClickDeleteRegistration(registration)}>
```

Comment:
`registration` brukes ikke til noe her
```suggestion
        <Button fullWidth variant="contained" onClick={() => setShowDeleteModal(true)>
```